src/app/core/models/tester-response.model.ts

```diff
@@ -0,0 +1,13 @@
+export class TesterResponse {
+    itemName: string; // e.g Change of Severity
```

Comment:
Can we change this to `responseTitle` ? And the `itemDescription` to `responseDescription`? Since this is the `TesterResponse` model like makes more sense to have the word `response` as opposed to `item`? What do you think? @KevinCJH

Reply:
I think we should leave it as `title` and `description` as its not a response.